pkg/install/disableupdates_test.go

```diff
@@ -0,0 +1,53 @@
+package install
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"testing"
+
+	v1 "github.com/openshift/api/config/v1"
+	configclient "github.com/openshift/client-go/config/clientset/versioned"
+	"github.com/openshift/client-go/config/clientset/versioned/fake"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func TestDisableUpdates(t *testing.T) {
+	ctx := context.Background()
+
+	versionName := "version"
+
+	i := &Installer{
+		configcli: fake.NewSimpleClientset(&v1.ClusterVersion{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: versionName,
+			},
+			Spec: v1.ClusterVersionSpec{
+				Upstream: "RemoveMe",
+				Channel:  "RemoveMe",
+			},
+		}),
+	}
+
+	err := i.disableUpdates(ctx)
+	if err != nil {
+		t.Error(err)
+	}
+
+	cv := getClusterVersion(t, i.configcli, versionName)
+	if cv.Spec.Upstream != "" {
+		t.Errorf("cv.Spec.Upstream was not cleared. Got %s, want empty string", cv.Spec.Upstream)
+	}
+	if cv.Spec.Channel != "" {
+		t.Errorf("cv.Spec.Channel was not cleared. Got %s, want empty string", cv.Spec.Channel)
+	}
+}
+
+func getClusterVersion(t *testing.T, configcli configclient.Interface, versionName string) *v1.ClusterVersion {
+	cv, err := configcli.ConfigV1().ClusterVersions().Get(versionName, metav1.GetOptions{})
+	if err != nil {
+		t.Error(err)
+	}
+	return cv
+}
```

Comment:
I don't see why a separate function is warranted for this?

Reply:
Fixed